pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -154,7 +154,6 @@ let
       curl
       libepoxy
       libffi
-    ] ++ lib.optionals (chromiumVersionAtLeast "114") [
```

Comment:
Removing these version checks made sense in the original commit, but in this branch `beta` is still on version 113, so I think we should either leave these checks in or also backport whatever commits bumped `beta`.

Reply:
Thanks, it should be fine though. Users shouldn't use the beta dev versions on stable NixOS releases anyway since we don't regularly backport those updates. And the newer chromiumBeta version (M115) is currently broken anyway. Let's merge this then as the security fixes for ungoogled-chromium are way more important than the optional chrome{Beta,Dev} builds. Good catch though - thanks for paying attention :)